README.md

```diff
@@ -1,2 +1,53 @@
-# UEPlugin-AdvancedControlFlow
-Unreal Engine Plugin: Advanced Control Flow 
+# Unreal Engine Plugin: Advanced Control Flow
+
+Advanced Control Flow is a Unreal Engine plugin which adds custom Blueprint nodes to realize the complex control flow.
+
+The typical use case of this plugin is when you try to realize the if-elseif-else statement.  
+To realize if-elseif-else statement on the vanilla Unreal Engine, you need to stack the multiple branch nodes to check multiple conditions.  
+By using this plugin, you can realize this case with only one MultiBranch node.
+
+![](docs/images/thumbnail/advanced_control_flow.png)
+
+## Features
+
+* MultiBranch
+  * Realize if-elseif-else statement (multiple conditional branches).
+* Conditional Sequence
+  * Execute all relevant execution pins if the conditional pin is true.
```

Comment:
"all relevant execution pins if the conditional pin is true" is correct?
 Each?                                     Each?